src/targets/services/aggregation.js

```diff
@@ -44,29 +39,35 @@ export const aggregation = async () => {
 
   // TODO: Remove hierarchy and base only on metadata and id
   // Count files in the aggregation folder
-  const { data: unfilteredFiles } = await client.query(Q('io.cozy.files').where({
-    dir_id: aggregationDirectory
-  }))
-  const receivedShares = unfilteredFiles.filter(file => file.attributes.metadata && (file.attributes.metadata.level === level))
+  const { data: unfilteredFiles } = await client.query(
+    Q('io.cozy.files').where({
+      dir_id: aggregationDirectoryId
+    })
+  )
+  const receivedShares = unfilteredFiles.filter(
+    file => file.attributes.metadata && file.attributes.metadata.level === level
+  )
 
-  console.log('Already received shares:', receivedShares.length)
+  log(infoTag, 'Already received shares:', receivedShares.length)
 
   if (receivedShares.length !== nbChild) {
-    console.log('Waiting for more...')
+    log(infoTag, 'Waiting for more...')
     return
   }
 
-  console.log('Received the right amount of shares, let\'s start')
+  log(infoTag, 'Received the right amount of shares, starting!')
 
   // Fetch all stored shares
   const shares = []
   for (let s of receivedShares) {
-    shares.push(
-      JSON.parse(await client.stackClient.fetchJSON('GET', `/files/download/${s.id}`))
+    const receivedShare = await client.stackClient.fetchJSON(
+      'GET',
+      `/files/download/${s.id}`
     )
```

Comment:
This could be replaced by `const receivedShare = await client.collection('io.cozy.files').download(s)` : https://github.com/cozy/cozy-client/blob/master/packages/cozy-stack-client/src/FileCollection.js#L518